### PR TITLE
Docs: Configuring external DNS for a custom domain

### PR DESCRIPTION
**Pull request by Mariana-Caetano**
## What's the purpose of this pull request?

This PR adds the second doc for the [Go-live: VTEX Integration track](https://evergreen.faststore.dev/docs/go-live/2-configuring-external-dns).

🔗 [Preview](https://faststore-site-git-docs-configuring-external-dns-faststore.vercel.app/docs/go-live/2-configuring-external-dns)

### CODE REVIEW

File: apps/site/pages/docs/go-live/2-configuring-external-dns.mdx [hunk 1]
```diff
@@ -7,10 +7,121 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
 
 <header>
 
-# Part 2: Configuring external DNS for a custom domain
+# 2. Configuring external DNS for a custom domain
```
Comment (Mariana-Caetano) [inline apps/site/pages/docs/go-live/2-configuring-external-dns.mdx:10]:
```suggestion
# Part 2: Configuring external DNS for a custom domain
```

**Comment by hellofanny** [comment]
It's missing this page, right?
image

File: apps/site/pages/docs/go-live/2-configuring-external-dns.mdx [hunk 2]
```diff
@@ -13,4 +13,115 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
 
 <Callout type="warning" emoji="⚠️">
 This documentation is currently under development.
-</Callout>
\ No newline at end of file
+</Callout>
+
+In this guide, you will learn how to configure external domains for your **FastStore + VTEX** website. The process involves the following steps:
+
+ - Set up your external DNS provider to point your custom domain to your VTEX website. Custom domains can be purchased from any **domain registrar**.
```
Comment (hellofanny) [inline apps/site/pages/docs/go-live/2-configuring-external-dns.mdx:20]:
```suggestion
 - Set up your external DNS provider to point your custom domain to your VTEX website. Custom domains can be purchased from any **domain registration platform**.
```

is that?

File: apps/site/pages/docs/go-live/2-configuring-external-dns.mdx [hunk 3]
```diff
@@ -13,4 +13,115 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
...
+ - Configure your custom domains in **VTEX Account Settings**.
+ - Associate your custom domains with your **FastStore** project. 
+
+Besides making your FastStore website accessible via a custom domain, the **VTEX Checkout** will also be operational as soon as you complete this guide. You'll then be ready to continue configuring other VTEX solutions, such as **Order Placed**, **Login**, and **My Account**.
+
+---
+
+## Before you start
+
+- [Set up your **VTEX** account.](/docs/go-live/1-setting-up-vtex-account-for-faststore)
```
Comment (hellofanny) [inline apps/site/pages/docs/go-live/2-configuring-external-dns.mdx:30]:
```suggestion
- [Set up your VTEX account.](/docs/go-live/1-setting-up-vtex-account-for-faststore)
```

File: apps/site/pages/docs/go-live/2-configuring-external-dns.mdx [hunk 4]
```diff
@@ -13,4 +13,115 @@ import { Callout, Tab, Tabs } from 'nextra-theme-docs'
...
+If your FastStore website is going live for the first time, [open a ticket with VTEX Support](https://help.vtex.com/en/support) and specify that you need to configure the CDN workflow for the secure and main domains of your store. Remember to include the following information in your ticket:
+
+- The account **name**.
+- The **secure** and **main** domains of your store.
+
+The VTEX team will evaluate your request and take the necessary actions to configure the CDN workflow for your account.
+
+---
+
+## Related resources
```
Comment (hellofanny) [inline apps/site/pages/docs/go-live/2-configuring-external-dns.mdx:124]:
```suggestion
## Related Resources
```

**Pull request merged**